sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py

```diff
@@ -192,7 +192,7 @@ def read_from_datastore(project, user_options, pipeline_options):
                                           num_shards=user_options.num_shards)
 
   # Actually run the pipeline (all operations above are deferred).
-  return p.run()
+  return p.run().wait_until_finish()
```

Comment:
Here, we return the result.  However, I don't think the result is actually used.  Should we remove "return" for consistency?